wolfcrypt/src/asn.c

```diff
@@ -9156,7 +9156,7 @@ static const ASNItem p8EncPbes1ASN[] = {
 /* ENCALGO_PBEPARAM_SALT */             { 3, ASN_OCTET_STRING, 0, 0, 0 },
                         /* Iteration Count */
 /* ENCALGO_PBEPARAM_ITER */             { 3, ASN_INTEGER, 0, 0, 0 },
-/* ENCDATA               */     { 1, ASN_OCTET_STRING, 0, 0, 0 },
```

Comment:
Do you mean CONTEXT_SPECIFIC?
PKCS#8 RFC 5958, 3 - EncryptedPrivateKeyInfo says it will be an OCTECT_STRING.

Why is OpenSSL different?

Reply:
Yes I believe you are right, CONTEXT_SPECIFIC is actually what I need here.

I completely agree that our encoding is exactly to specification. I do not know why OpenSSL does things differently, but its also how the original code before WOLFSSL_ASN_TEMPLATE worked as well.